docs/rofl/prerequisites.md

```diff
@@ -0,0 +1,121 @@
+---
+description: How to build your first ROFL app
+---
+
+# Prerequisites
+
+This chapter will show you how to install the software required for developing
+a ROFL app using the Oasis SDK. After successfully completing all the described
+steps you will be able to start building your first ROFL app!
+
+If you already have everything set up, feel free to skip to the [next chapter].
+
+[next chapter]: app.md
+
+## Environment Setup
+
+The following is a list of prerequisites required to start developing using the
+Oasis SDK:
+
+### [Rust]
+
+We follow [Rust upstream's recommendation][rust-upstream-rustup] on using
+[rustup] to install and manage Rust versions.
+
+:::info
+
+rustup cannot be installed alongside a distribution packaged Rust version. You
+will need to remove it (if it's present) before you can start using rustup.
+
+:::
+
+Install it by running:
+
+```bash
+curl --proto '=https' --tlsv1.2 -sSf https://sh.rustup.rs | sh
+```
+
+:::info
```

Comment:
Info and other box messages in dark mode really catch your attention, which is strange, as the writer should encourage the reader to use the install procedure written in the paragraph, and not the alternative in the box. Just an observation.